checkers-flex.html

```diff
@@ -87,15 +90,95 @@
     </div>
     <!--1st row -->
     <div class="row">
-      <div class="box black-square" id="a1" nextValidSquare="['b2']"></div>
+
+      <div class="box black-square" id="a1" nextValidSquare="['b2']">
+        <div class="piece black" :id="player1.chips[0].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="c1" nextValidSquare="['b2', 'd2']"></div>
+
+      <div class="box black-square" id="c1" nextValidSquare="['b2', 'd2']">
+        <div class="piece black" :id="player1.chips[1].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="e1" nextValidSquare="['d2', 'f2']"></div>
+
+      <div class="box black-square" id="e1" nextValidSquare="['d2', 'f2']">
+        <div class="piece black" :id="player1.chips[2].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="g1" nextValidSquare="['f2', 'h2']"></div>
+
+      <div class="box black-square" id="g1" nextValidSquare="['f2', 'h2']">
+        <div class="piece black" :id="player1.chips[3].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
     </div>
   </div>
+</div>
+
+  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+  <script>
+
+    new Vue({
+      el: '#game',
+      // data: chips :: when the chips_state.json can be imported, we can store it in a variable, such as `chips`, but for this part I will hardcode the data...
+      data: {
```

Comment:
eventually when we break this all up into `.vue` templates with imported data, you will see how this all can be broken down and then compiled automatically, but for now as we figure out the nuances of our data needs, we can play with the hardcoded object here and backfill changes to the `chips_state.json` file.